packages/preview/modern-ysu-thesis/0.1.0/template/thesis.typ

```diff
@@ -0,0 +1,224 @@
+#import "@preview/modern-ysu-thesis:0.1.0": documentclass, indent
+
+#let (
+  // 布局函数
+  twoside, doc, preface,page-header, mainmatter, mainmatter-end, appendix,
+  // 页面函数
+  fonts-display-page, cover,detail, decl-page, abstract, abstract-en, bilingual-bibliography,
+  outline-page, list-of-figures, list-of-tables, notation, acknowledgement,
+) = documentclass(
+  // doctype: "bachelor",  // "bachelor" | "master" | "doctor" | "postdoc", 文档类型，默认为本科生 bachelor
+  // degree: "academic",  // "academic" | "professional", 学位类型，默认为学术型 academic
+  // anonymous: true,  // 盲审模式
+  twoside: false,  // 双面模式，会加入空白页，便于打印
+  // 可自定义字体，先英文字体后中文字体，应传入「宋体」、「黑体」、「楷体」、「仿宋」、「等宽」
+  // fonts: (楷体: ("Times New Roman", "FZKai-Z03S")),
+  info: (
+    title: ("不是，哥们，这登龙能空的啊"),
+    title-en: "My Title in English",
+    grade: "20XX",
+    student-id: "1234567890",
+    author: "张三",
+    author-en: "Ming Xing",
+    department: "某学院",
+    department-en: "School of Chemistry and Chemical Engineering",
+    major: "某专业",
+    major-en: "Chemistry",
+    supervisor: ("李四", "教授"),
+    supervisor-en: "Professor My Supervisor",
+    // supervisor-ii: ("王五", "副教授"),
+    // supervisor-ii-en: "Professor My Supervisor",
+    submit-date: datetime.today(),
+  ),
+  // 参考文献源
+  bibliography: bibliography.with("ref.bib"),
+)
+
+// 文稿设置
+#show: doc
+
+// 字体展示测试页
+// #fonts-display-page()
+
+// 封面页
+#cover()
+
+#detail()
+// 声明页
+#decl-page()
+#show: preface
+
+// 前言
+// 中文摘要
+#abstract(
+  keywords: ("我", "就是", "测试用", "关键词","需要","更多","更多","更多","更多","更多","关键词","来测试")
+)[
+  中文摘要
+]
+
+// 英文摘要
+#abstract-en(
+  keywords: ("Dummy", "Keywords", "Here", "It Is")
+)[
+  English abstract
+]
+
+
+// 目录
+#outline-page()
+
+// 插图目录
+// #list-of-figures()
+
+// 表格目录
+// #list-of-tables()
+
+// 正文
+#show: mainmatter
+// #show: page-header
+
+// 符号表
+// #notation[
+//   / DFT: 密度泛函理论 (Density functional theory)
+//   / DMRG: 密度矩阵重正化群密度矩阵重正化群密度矩阵重正化群 (Density-Matrix Reformation-Group)
+// ]
+
+= 绪　论
+== 列表
+=== 无序列表
+==== asd
+asdasd
+
+asdasda
+
+asdasd
+- 无序列表项一
+- 无序列表项二
+  - 无序子列表项一
+  - 无序子列表项二
+
+=== 有序列表
+
++ 有序列表项一
++ 有序列表项二
++ 表3
+  + 有序子列表项一
+  + 有序子列表项二
+
+=== 术语列表
+
+/ 术语一: 术语解释
+/ 术语二: 术语解释
+
+=== temp
+== 图表
+
+引用@tbl:timing，引用@tbl:timing-tlt，以及@fig:ysu-logo。引用图表时，表格和图片分别需要加上 `tbl:`和`fig:` 前缀才能正常显示编号。
+
+#align(center, (stack(dir: ltr)[
+  #figure(
+    table(
+      align: center + horizon,
+      columns: 4,
+      [t], [1], [2], [3],
+      [y], [0.3s], [0.4s], [0.8s],
+    ),
+    caption: [常规表],
+  ) <timing>
+][
+  #h(50pt)
+][
+  #figure(
+    table(
+      columns: 4,
+      stroke: none,
+      table.hline(),
+      [t], [1], [2], [3],
+      table.hline(stroke: .5pt),
+      [y], [0.3s], [0.4s], [0.8s],
+      table.hline(),
+    ),
+    caption: [三线表],
+  ) <timing-tlt>
+]))
+
+#figure(
+  image("..\assets\vi\ysulogo.png", width: 20%),
```

Comment:
This won't work, as this file will not be copied when creating a project from this template. If you consider that this logo is important enough to be used by most people who use your package, you can export it from the package itself by adding this in `lib.typ` (or some other file, and re-exporting it here):

```typst
#let logo = image.with("assets/vi/ysulogo.png")
```

And then, in your template, you can do:

```typst
#logo(width: 20%)
```

Reply:
ok,i am sorry that i am busy in do my thesis and no see this reply ,i will fix these as quick as i can